webpack/InsightsCloudSync/InsightsCloudSyncSelectors.js

```diff
@@ -0,0 +1,4 @@
+// export const selectForemanInventoryUpload = state => state.ForemanInventoryUpload || {};
+// Added 'state' to use redux in storybook only, should be removed.
+export const selectInsightsCloudSync = state =>
+  state.InsightsCloudSync || state || {};
```

Comment:
can you move the selector to https://github.com/theforeman/foreman_rh_cloud/blob/master/webpack/ForemanRhCloudSelectors.js#L2